tests/unit/__init__.py

```diff
@@ -33,7 +31,9 @@
 SOURCE_DIR = os.path.join(TOP_DIR, "source")
 # Let us import modules from the NVDA source.
 sys.path.insert(1, SOURCE_DIR)
-import sourceEnv
+import sourceEnv  # noqa: F401

```

Comment:
I'm fine with using `noqa` to suppress this case, however please include a comment about why and a hint about what the lint error is. It's not much fun for a future developer to have to look these up and understand why it is like this.

Reply:
What would be proper syntax for this? I assume something like this is not going to work

```suggestion
import sourceEnv  # noqa: F401  module imported but unused
```

Reply:
Yep something like that works, though it gets messy if you have multiple errors to ignore. 

I would do:
```
# F401: module imported but unused. We need sourceEnv for unit tests because of ?
import sourceEnv  # noqa: F401
```

Reply:
I think I've done something along those lines.